internal/storage/v1/elasticsearch/spanstore/readerv1_test.go

```diff
@@ -54,8 +54,8 @@ func TestSpanReaderV1_FindTraces(t *testing.T) {
 		spanID2 := model.NewSpanID(2)
 		trace1 := getTestingTrace(traceID1, spanID1)
 		trace2 := getTestingTrace(traceID2, spanID2)
-		traces := []*dbmodel.Trace{trace1, trace2}
-		m.On("FindTraces", mock.Anything, mock.AnythingOfType("*dbmodel.TraceQueryParameters")).Return(traces, nil)
+		traces := []dbmodel.Trace{trace1, trace2}
+		m.On("FindTraces", mock.Anything, mock.AnythingOfType("dbmodel.TraceQueryParameters")).Return(traces, nil)
```

Comment:
when you are not matching on the content of the parameters, there's no benefit to using `AnythingOfType`, just use `mock.Anything`